api/app/signals/apps/api/templates/api/swagger/openapi.yaml

```diff
@@ -967,6 +967,131 @@ paths:
       - OAuth2:
           - SIG/ALL
 
+  /signals/v1/private/expressions/:
```

Comment:
In general this could be a bit more descriptive

Reply:
We would like the following:
* mark as experimental
* explain what an expression is